app/tasks/raster_tile_cache_assets/symbology.py

```diff
@@ -48,6 +47,46 @@ async def no_symbology(
         raise RuntimeError("No source URI set.")
 
 
+async def create_colormap_symbology_job(
```

Comment:
why pull this code block into a seperate function? Is it a reused else where?
To me it looks like a continuation of the `pixetl_symbology` function. The actual job only gets generated in the gdaldemjob function, right?